src/api_calls/orders_calls.js

```diff
@@ -75,7 +75,7 @@ export function deleteOrder(id){
 
 }
 
-export function updateOrder(id,patchRequestBody){
+export async function updateOrder(id,patchRequestBody){
```

Comment:
It is not necessary to declare these function async if you do not use await in function block.